### DISCUSSION TITLE
FutureCall for db backup

### DISCUSSION THREAD

**Discussion by sulthanalihsan** [serverpod/serverpod#1182] [Q&A] [answered]
I'm interested in using the future call feature in Serverpod. I would like to know if the future call feature can be utilized to schedule automatic database backup tasks. If it is possible, how to set it up?

**Comment by Isakdl** [comment, marked as answer]
I suppose you could in theory, but I think these types of things should live outside your own code. You will most likely need the backup if something goes horribly wrong, the most likely point of failure will be within your own code, hence not relying on your own code to make sure backups are created is safer.

If you are using a managed sql database such as the one on [GCP](https://cloud.google.com/sql/#section-2) you get backups automatically out of the box without any additional work.
